tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -0,0 +1,434 @@
+
+		$before = tribe_get_option( $this->opts_prefix . 'content_after' );
+		if ( ! empty( $before ) ) {
+			add_filter( 'tribe_events_eventbrite_before_the_tickets', array( $this, 'before_iframe' ) );
+		}
+
+		$after = tribe_get_option( $this->opts_prefix . 'content_after' );
+		if ( ! empty( $after ) ) {
+			add_filter( 'tribe_events_eventbrite_after_the_tickets', array( $this, 'after_iframe' ) );
+		}
+
+		$show_tickets_private_events = tribe_get_option( $this->opts_prefix . 'show_tickets_private_events' );
+		if ( ! empty( $show_tickets_private_events ) ) {
+			add_filter( 'tribe_events_eventbrite_the_tickets', array( $this, 'render_tickets_iframe_private_eb_events' ) );
+		}
+
+		add_filter( 'tribe_events_eb_iframe_html', array( $this, 'iframe_height' ), 10, 3 );
+
+		if ( 'en_US' !== $this->get_chosen_eb_locale() ) {
+			add_filter( 'tribe-eventbrite-base_api_url', array( $this, 'api_url' ) );
+
+			add_filter( 'tribe_eb_api_sync_event', array( $this, 'eb_currency_for_update_create' ), 10, 5 );
+		}
+
+		// originally from https://theeventscalendar.com/knowledgebase/use-the-event-cost-field-with-tickets-installed/
+		// cannot use https://github.com/moderntribe/products-extensions/blob/master/tribe-ext-show-cost-field/index.php because that Extension requires Event Tickets Plus active
+		$show_admin_cost_field = tribe_get_option( $this->opts_prefix . 'show_admin_cost_field' );
+		if ( ! empty( $show_admin_cost_field ) ) {
+			add_filter( 'tribe_events_admin_show_cost_field', '__return_true', 100 );
+		}
+
+		// originally from https://theeventscalendar.com/knowledgebase/moving-the-eventbrite-tickets-box/
+		$eb_new_ticket_location = tribe_get_option( $this->opts_prefix . 'ticket_single_location' );
+		$eb_ticket_default_location = 'tribe_events_single_event_after_the_meta';
+		if ( ! empty( $eb_new_ticket_location ) && $eb_ticket_default_location !== $eb_new_ticket_location ) {
+			$display_tickets = array( tribe( 'eventbrite.main' ), 'print_ticket_form' );
+			remove_action( $eb_ticket_default_location, $display_tickets, 9 );
+			add_action( $eb_new_ticket_location, $display_tickets );
+		}
+	}
+
+	public function before_iframe() {
+		$text = tribe_get_option( $this->opts_prefix . 'content_before' );
+
+		$text = sprintf( '<h3 class="tribe_ext_eventbrite_opts">%s</h3>', $text );
+
+		return $text;
+	}
+
+	public function after_iframe() {
+		$text = tribe_get_option( $this->opts_prefix . 'content_after' );
+
+		$text = sprintf( '<span class="tribe_ext_eventbrite_opts">%s</span>', $text );
+
+		return $text;
+	}
+
+	// originally from https://theeventscalendar.com/knowledgebase/changing-the-height-of-the-eventbrite-tickets-iframe/
+	public function iframe_height( $html, $event_id, $post_id ) {
+		// should only be a positive integer, per validation
+		$new_height = intval( tribe_get_option( $this->opts_prefix . 'iframe_px' ) );
+
+		if ( 200 === $new_height ) {
+			return $html;
+		}
```

Comment:
What if we change the default in the core plugin, we'll have to update it here too or it will misbehave. Let's remove this entire block as its unncessary and will require maintenance:
```
if ( 200 === $new_height ) {
	return $html;
}
```